components/Tests/AddTwoIntegers.tsx

```diff
@@ -0,0 +1,38 @@
+import {CandidateTestModel, GetCandidateTestModel} from "../../Models/CandidateTestModel";
+import InstructionsBlock, {
+    GetInputOutputProps,
+    GetInstructionsProps,
+    InputOutputProps,
+    InstructionsProps
+} from "../Instructions/Instructions";
+import {CandidateTest} from "../ApiFetch/TestFetcher";
+import {GetTest} from "./TestEditorBuilder";
+
+const Number = 1;
+const Title = "Add Two Integers";
+const Brief = "You are given a two-digit integer n. Return the sum of its digits.";
+const Example = "For n = 29, the output should be\n" +
+    "addTwoDigits(n) = 11.";
+const ModelAnswer = "function addTwoDigits(n) {\n}";
+const Input: any = 29;
+const ExpectedOutput: any = 11;
+const JsTips = "// Prints help message to the console\n" +
+    "// Returns a string\n" +
+    "function helloWorld(name) {\n" +
+    "    console.log(\"This prints to the console when you Run Tests\");\n" +
+    "    return \"Hello, \" + name;\n" +
+    "}";
+const ExecutionTimeLimit = "4 seconds";
+const InputType = "integer n";
+const InputDescription = "A positive two-digit integer.\n" +
+    "Guaranteed constraints:\n" +
+    "10 ≤ n ≤ 99.";
+const OutputType = "integer";
+const OutputDescription = "The sum of the first and second digits of the input number.";
```

Comment:
do you think all of this could be covered more easily in the Brief and Example?

Reply:
Could do but wanted to reflect the layout of the test instructions as Sasha gave them to us:
You are given a two-digit integer n. Return the sum of its digits.
Example
For n = 29, the output should be
addTwoDigits(n) = 11.
Input/Output
•	[execution time limit] 4 seconds (js)
•	[input] integer n
A positive two-digit integer.
Guaranteed constraints:
10 ≤ n ≤ 99.
•	[output] integer
o	The sum of the first and second digits of the input number.
[JavaScript (ES6)] Syntax Tips
// Prints help message to the console
// Returns a string
function helloWorld(name) {
    console.log("This prints to the console when you Run Tests");
    return "Hello, " + name;
}
 
 
 
TEST 1
Input:
n: 29
Expected Output:
 
TEST 2
Input:
n: 48
Expected Output:
12
 
TEST 3
Input:
n: 10
Expected Output:
 
TEST 4
Input:
n: 25
Expected Output:
 
TEST 5 
Input:
n: 52
Expected Output: 7